rubygem/lib/zeus/rails.rb

```diff
@@ -182,6 +182,14 @@ def test_environment
     end
 
     def test_helper
+      # don't let minitest setup another exit hook
+      begin
+        require 'minitest/unit'
+        MiniTest::Unit.class_variable_set("@@installed_at_exit", true)
```

Comment:
FWIW, [minitest-line](https://github.com/judofyr/minitest-line) can be used to run a test by line number in Minitest 5.0+.